internal/docker/handler_ssh.go

```diff
@@ -1,7 +1,15 @@
 package docker
 
 import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/containerssh/libcontainerssh/config"
+	"github.com/containerssh/libcontainerssh/internal/agentforward"
 	"github.com/containerssh/libcontainerssh/internal/sshserver"
+	"github.com/containerssh/libcontainerssh/message"
+	"golang.org/x/crypto/ssh"
```

Comment:
The backends shouldn't depend directly on this library as we may want to replace it fairly soon. Maybe we need to proxy something?

Reply:
The only thing that's there from the library is the rejection message type. Should pose no problem to replace it when we decide to.